Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# South Seas and Ulduar model  positions

## Developer changelog:
- Fixed #1328, broken fleet model positions near Pandaria (spaghetti on the map when you sent fleet there).
- Fixed #1334, model positions near Ulduar.


# How to test:
Try to send your fleet there.
To understand how it looks, check screenshots here: https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/issues/1328.
